test/setup.js

```diff
@@ -0,0 +1,30 @@
+import GraphQLJSClient from '../src/graphql-client';
+
+function recordTypes() {
+  const types = GraphQLJSClient.trackedTypes();
+
+  if (typeof require === 'function') {
+    const body = JSON.stringify({'profiled-types': types}, null, '  ');
```

Comment:
Don't care either way, but I'm just used to seeing a number for the last arg here: `2`